internal/cmd/beta/network-area/create/create.go

```diff
@@ -0,0 +1,203 @@
+	rmUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/resourcemanager/utils"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	nameFlag                = "name"
+	organizationIdFlag      = "organization-id"
+	dnsNameServersFlag      = "dns-name-servers"
+	networkRangesFlag       = "network-ranges"
+	transferNetworkFlag     = "transfer-network"
+	defaultPrefixLengthFlag = "default-prefix-length"
+	maxPrefixLengthFlag     = "max-prefix-length"
+	minPrefixLengthFlag     = "min-prefix-length"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	Name                *string
+	OrganizationId      *string
+	DnsNameServers      *[]string
+	NetworkRanges       *[]string
+	TransferNetwork     *string
+	DefaultPrefixLength *int64
+	MaxPrefixLength     *int64
+	MinPrefixLength     *int64
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates a network area",
+		Long:  "Creates a network area in an organization.",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`Create a network area with name "network-area-1" in organization with ID "xxx" with network ranges and a transfer network`,
+				`$ stackit beta network-area create --name network-area-1 --organization-id xxx --network-ranges "1.1.1.0/24,192.123.1.0/24" --transfer-network "192.160.0.0/24"`,
+			),
+			examples.NewExample(
+				`Create a network area with name "network-area-2" in organization with ID "xxx" with network ranges, transfer network and DNS name server`,
+				`$ stackit beta network-area create --name network-area-2 --organization-id xxx --network-ranges "1.1.1.0/24,192.123.1.0/24" --transfer-network "192.160.0.0/24" --dns-name-servers "1.1.1.1"`,
+			),
+			examples.NewExample(
+				`Create a network area with name "network-area-3" in organization with ID "xxx" with network ranges, transfer network and additional options`,
+				`$ stackit beta network-area create --name network-area-3 --organization-id xxx --network-ranges "1.1.1.0/24,192.123.1.0/24" --transfer-network "192.160.0.0/24" --default-prefix-length 25 --max-prefix-length 29 --min-prefix-length 24`,
+			),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			var orgLabel string
+			rmApiClient, err := rmClient.ConfigureClient(p)
+			if err == nil {
+				orgLabel, err = rmUtils.GetOrganizationName(ctx, rmApiClient, *model.OrganizationId)
+				if err != nil {
+					p.Debug(print.ErrorLevel, "get organization name: %v", err)
+					orgLabel = *model.OrganizationId
+				}
+			} else {
+				p.Debug(print.ErrorLevel, "configure resource manager client: %v", err)
+			}
```

Comment:
Wouldn't it be simpler to have something like this?

```suggestion
rmApiClient, err := rmClient.ConfigureClient(p)
if err != nil {
	p.Debug(print.ErrorLevel, "configure resource manager client: %v", err)
} 
orgLabel, err := rmUtils.GetOrganizationName(ctx, rmApiClient, *modelOrganizationId)
if err != nil {
	p.Debug(print.ErrorLevel, "get organization name: %v", err)
	orgLabel = *model.OrganizationId
}
```

Reply:
Hmm yes, its simpler. But if the configuration of the client errors, the `GetOrganizationName` call won't work either and the user will get 2 errors, when it's really just the one about the client

Reply:
Ah yes we don't want to return earlier